index.js

```diff
@@ -1310,6 +1311,28 @@
   }
   S.fromMaybe = def('fromMaybe', {}, [a, $Maybe(a), a], fromMaybe);
 
+  //# fromMaybe_ :: (() -> a) -> Maybe a -> a
+  //.
+  //. Variant of [`fromMaybe`](#fromMaybe) which takes a thunk so the default
+  //. value is only computed if required.
+  //.
+  //. ```javascript
+  //. > global.fib = function fib(n) {
+  //. .   return n <= 1 ? n : fib(n - 2) + fib(n - 1);
+  //. . }
+  //. fib
```

Comment:
I had to read this three times before understanding that you're *just* using it as an example of a computationally expensive function. The fact that you explicitly state the return value of the function definition, and that you're explicitly attaching it to the global scope, makes no sense without knowledge of `doctest`. It seemed to me at first as if you're trying to demonstrate something here, whereas in fact you're really just working around the semantics of your linter.

I think it might be better to use something native to JavaScript (or in fact exposed by Sanctuary) to demonstrate. Maybe even something like `/*perform expensive calculation*/ 1 + 1`. Or if you really feel `fib` demonstrates it best, it would've helped me to see a comment above fibs definition explaining me why it's being defined there.